openshift/dataverse-broker.yaml

```diff
@@ -10,6 +10,14 @@ objects:
     name: dataverse-broker
   spec:
     url: http://dataverse-broker.bdc.svc.cluster.local
+    insecureSkipTLSVerify: false
+    caBundle: ${CA_BUNDLE}
```

Comment:
You can remove the `caBundle` and field for now - you don't need this until your broker serves https.
You should also remove `insecureSkipVerify`.